<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transferencias</class>
 <widget class="QDialog" name="Transferencias">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(94, 92, 100);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tlwg Typewriter</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Deposito bancario Internacional</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="tipoDocumento">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>-Tipo de documento</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DNI - Documento nacional de identidad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CUIT / CUIL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pasaporte</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="valorDocumento">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Monto</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="montoTransferencia">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="tipoMoneda">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>-Seleccione el tipo de moneda*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dolares</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Euro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Crypto</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnRegistroDeposito">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nombre*</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombreCompleto">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sexo*</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese CBU*</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCbu">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="txtSexo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>-Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Masculino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Femenino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Otro</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email*</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmail">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ciudad*</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="txtCiudad">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>-Seleccione una opcion</string>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apellido*</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtApellido">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="txtMotivo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>-Seleccione el motivo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alquiler</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pago de haberes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Expensas</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Otros</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Motivo*</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images/routeImagenes.qrc"/>
 </resources>
 <connections/>
</ui>
